### ISSUE TITLE
Error installing pyodbc 4.0.30 on Max OS X

### ISSUE THREAD

**Issue by srossgupta** [mkleehammer/pyodbc#711]
### Environment

- Python: 3.7.6
- pyodbc: 4.0.30
- OS: Mac OS Catalina
- DB:
- driver:

### Issue
Unable to find matching distribution corresponding to Pyodbc 4.0.30. There are no files published for Mac OSX for 4.0.30.
https://pypi.org/project/pyodbc/4.0.30/#files
```
[pipenv.exceptions.InstallError]: ['ERROR: Could not find a version that satisfies the requirement pyodbc==4.0.30 (from -r /tmp/pipenv-37pca4lx-requirements/pipenv-exfib4dx-requirement.txt (line 1)) (from versions: 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.6, 2.1.7, 2.1.8, 2.1.9, 2.1.10, 2.1.11, 3.0.3, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.1.1, 4.0.1, 4.0.2, 4.0.3, 4.0.5, 4.0.6, 4.0.7, 4.0.8, 4.0.9, 4.0.10, 4.0.11, 4.0.12, 4.0.13, 4.0.14, 4.0.15, 4.0.16, 4.0.17, 4.0.18, 4.0.19, 4.0.20, 4.0.21, 4.0.22, 4.0.23, 4.0.24, 4.0.25, 4.0.26, 4.0.27, 4.0.28, 4.0.30b1)', 'ERROR: No matching distribution found for pyodbc==4.0.30 (from -r /tmp/pipenv-37pca4lx-requirements/pipenv-exfib4dx-requirement.txt (line 1))']
```
Often it is easiest to describe your issue as "expected behavior" and "observed behavior".

**Comment by keitherskine** [comment]
The MacOSX 4.0.30 files were uploaded separately and can be found at:
https://pypi.org/project/pyodbc/4.0.30b1/#files

**Comment by ribeaud** [comment]
I got the same problem and NOT using a **Mac** by running `pipenv update`. My **ODBC** dependency is declared as following:

    pyodbc = '>=4.0.28'

And, by the way, **AppVeyor** is throwing a build failing at https://pypi.org/project/pyodbc/. Important?

**Comment by srossgupta** [comment]
I am hitting the same error while trying to do `pipenv update`. My dependency is defined as 
```
pyodbc = '*'
```
The problem is when the Pipfile.lock is generated it is picking a version 4.0.30 but that no longer exists and should be 4.0.30b1 . We have a solution of casting it to an older version 4.0.28 but requires multiple changes because of multiple repos using this same library

**Comment by keitherskine** [comment]
@mkleehammer just a quick heads-up to make sure you were aware of these issues.

**Comment by keitherskine** [comment]
@srossgupta After the recent fix for #712 , is this still an issue?

**Comment by mkleehammer** [comment]
Since 4.0.30 wasn't in the list of available versions I'm going to assume that #712 fixed it. (Thanks for notifying me of that Keith.). Please reopen if it is still an issue.
